packages/cloud-cognitive/src/components/Datagrid/utils/makeData.js

```diff
@@ -87,6 +95,30 @@ const newPerson = () => {
         : statusChance > 0.33
         ? yesterdayDate
         : twoDaysAgoDate,
+    bonus: `$\r${getRandomInteger(100, 500, 2)}`,
+    status_icon:
```

Comment:
minor thing, but this feels a little redundant. i think it would be cleaner to have a helper function to get the props for the status icon. something like
```
const getStatus = () => {
const props = { size: 'sm', theme: 'light' }
if (statusChance > .66 ) {
 return { ...props, kind: 'critical', iconDescription: 'critical'}
}
...
}
...
const iconProps = getProps();
...
<StatusIcon {...iconProps} />
```
and this way you avoid all the conditional renders